app/connectors/CdsFileUploadConnector.scala

```diff
@@ -47,4 +47,16 @@ class CdsFileUploadConnector @Inject()(appConfig: AppConfig, httpClient: HttpCli
   def getDeclarationStatus(mrn: MRN)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     httpClient.GET[HttpResponse](cdsFileUploadConfig.fetchDeclarationStatusEndpoint(mrn.value))
 
+  def getVerifiedEmailAddress(eori: EORI)(implicit hc: HeaderCarrier): Future[Option[VerifiedEmailAddress]] =
+    httpClient
+      .GET[Option[VerifiedEmailAddress]](cdsFileUploadConfig.fetchVerifiedEmailEndpoint(eori.value))
+      .map { maybeVerifiedEmail =>
+        maybeVerifiedEmail match {
+          case Some(verifiedEmailAddress) =>
+            logger.debug(s"Found verified email for eori: $eori")
+          case None =>
+            logger.info(s"No verified email for eori: $eori")
+        }
```

Comment:
Uhm... is this logging really useful? In production the pattern matching block will still be executed but not being useful at all.

Reply:
Well I reused the same method that is used for logging retrieved notifications (above it). I thought it might be useful in the non-prod environments when a user is unexpectedly either redirected to the email front end (or not redirected when expecting to be).

Reply:
Honestly, I think the logging here is not useful at all. The eori is passed via url, accordingly in case of "Not found" you will have all info you need in Kibana due to the implicit MDTP logging.
At the end not a big deal btw.